Userland/Libraries/LibJS/Runtime/Object.cpp

```diff
@@ -190,6 +190,18 @@ bool Object::create_data_property_or_throw(PropertyName const& property_name, Va
     return success;
 }
 
+// 7.3.6 CreateNonEnumerableDataPropertyOrThrow ( O, P, V ), https://tc39.es/proposal-error-cause/#sec-createnonenumerabledatapropertyorthrow
+bool Object::create_non_enumerable_data_property_or_throw(PropertyName const& property_name, Value value)
+{
+    VERIFY(!value.is_empty());
+
+    // 1. Let newDesc be the PropertyDescriptor { [[Value]]: V, [[Writable]]: true, [[Enumerable]]: false, [[Configurable]]: true }.
+    PropertyDescriptor new_description = { .value = value, .writable = true, .enumerable = false, .configurable = true };
```

Comment:
Don't know why, but lately I prefer this style:

```suggestion
    auto new_description = PropertyDescriptor { .value = value, .writable = true, .enumerable = false, .configurable = true };
```

I guess it just looks so similar :D
Just a note, no action required (unless you happen to agree)

Reply:
done :)